test/runtests.jl

```diff
@@ -18,16 +17,16 @@ include("utils.jl")
     @test string(rsem) == "$(typeof(rsem))(2, pid=$(myid()))"
 
     try
-        @info("Expect \"ERROR (unhandled task failure)\" on Julia 0.6 only")
         asynctimedwait(1.0; kill=true) do
             release(rsem)
         end
         @test "Expected error but no error thrown" == nothing
     catch err
         @test err isa RemoteException
-        @test err.captured.ex isa AssertionError
+        expected = VERSION >= v"1.2.0-DEV.28" ? ErrorException : AssertionError
```

Comment:
What is the text of the ErrorException? Since these error types are very nonspecific we should check the text of the error probably.

Reply:
The text is "release count must match acquire count." I'll test for that specifically.